mappings/17w46a#17w47a.tinydiff

```diff
@@ -4740,7 +4740,7 @@ c	net/minecraft/unmapped/C_2289462
 		p	2			p_2
 		p	3			p_3
 		p	4			p_4
-c	net/minecraft/unmapped/C_3077593	net/minecraft/item/WoolItem	
+c	net/minecraft/unmapped/C_3077593	net/minecraft/item/ColoredBlockItem	
```

Comment:
This is a good change here, but in early versions (at least in Beta, up to maybe 1.7?) this class is explicitly `WoolItem`. The same goes for `ColoredBlock` (which ig needs fixing too then).

Reply:
how do i fix it?

Reply:
You have to find the first version where it needs to be fixed, change the mapping there, then save the change into the graph with either `saveMappingsUp` or `saveMappingsDown` (whichever is appropriate).